File: UT10/WebServicesExamples/app/src/main/java/alvarogonzalez/webservices/MainActivity.java
package alvarogonzalez.webservices;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.builtInButton).setOnClickListener( (View v)-> {
            startActivity(new Intent(this, ConexionBuiltInActivity.class));
        });

        findViewById(R.id.volleyButton).setOnClickListener( (View v)-> {
            startActivity(new Intent(this, ConexionVolleyActivity.class));
        });

        findViewById(R.id.retrofitConListaButton).setOnClickListener( (View v)-> {
            startActivity(new Intent(this, ConexionRetrofitActivity.class));
        });

        findViewById(R.id.retrofitConPathButton).setOnClickListener( (View v)-> {
            startActivity(new Intent(this, ConexionRetrofitConPathActivity.class));
        });

    }
}

File: UT08/MotosSinFragments/app/src/main/java/agonzalez/motos/MainActivity.kt
package agonzalez.motos

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        findViewById<Button>(R.id.sinFragmentsSutton).setOnClickListener{
            startActivity( Intent(this, ActivitySinFragments::class.java) )
        }
        findViewById<Button>(R.id.swipeButton).setOnClickListener{
            startActivity( Intent(this, ActivitySwipe::class.java) )
        }
        findViewById<Button>(R.id.tabsButton).setOnClickListener{
            startActivity( Intent(this, ActivityTabbed::class.java) )
        }
    }
}